### DISCUSSION TITLE
Progress on revamped taskbar10

### DISCUSSION THREAD

**Discussion by Nicolasdavinci08** [valinet/ExplorerPatcher#3359] [Q&A]
Hi there, how has progress on the revamped taskbar10 going? I remember seeing some photos of a copilot button on the taskbar and I wonder how development of it is going. :)

**Comment by Amrsatrio** [comment]
![image](https://github.com/valinet/ExplorerPatcher/assets/20662640/5c1d1ed3-d560-43d6-b5ed-cb7f928efc2f)

The task band has been reimplemented. It is now being tested by myself and by others in private. There are still some issues like the icons for UWP apps are not loaded.

Excluding multi monitor taskbar, it is 90% complete. Though about ETA I'm afraid that I cannot make an ETA. When it is released, it will be mentioned in the release notes.

**Comment by Amrsatrio** [comment]
@Nicolasdavinci08 I have released EP's taskbar implementation for 226xx. Check out the latest prerelease.
